### PR TITLE
Adding my haiku!

### PR DESCRIPTION
**Pull request by doc**
Yay Hacktoberfest.

### CODE REVIEW

**Review by ltagliaferri — CHANGES_REQUESTED** [review]
Thank you for submitting this haiku! Can you add line breaks at the end of each line, as detailed in the [line breaks section of the contributing guidelines](https://github.com/do-community/cloud_haiku#line-breaks). Your haiku lines will look like so:

```
My first haiku line<br>
```

Once you update, we can merge this in, thanks!

*Please note: after more than two days of inactivity, pull requests are subject to being closed and marked invalid by maintainers.*

**Comment by doc** [comment]
Hi @ltagliaferri, thanks for the review. I've pushed a version with the line breaks as requested.

Many thanks!

**Review by ltagliaferri — APPROVED** [review]
Thank you for this haiku and for contributing to open source 🎉

**Pull request merged**
